routes/api.mjs

```diff
@@ -48,4 +49,20 @@ router.post("/db/add/", async (req, res) => {
   });
 });
 
+router.post("/get/menu", async (req, res) => {
+  try {
+    return res.json(await getMenu(req.body));
+  } catch (e) {
+    console.error(e);
+  }
+});
+
+router.post("/get/menu", async (req, res) => {
```

Comment:
It looks like you have two functions that both use the same endpoint `/get/menu` there should only be one.

Reply:
Fixed